Hymns/Common/BottomSheetView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+private enum Constants {
+    static let radius: CGFloat = 16
+    static let indicatorHeight: CGFloat = 6
+    static let indicatorWidth: CGFloat = 60
+    static let snapRatio: CGFloat = 1
+    static let minHeightRatio: CGFloat = 1
```

Comment:
These seem to be unused. What are these for?

Reply:
Lol, yeah leftover code from the old bottom sheet view that would slide up and down with an indicator. No longer needed.

Reply:
Addressed in commit 3678e3a